Tools/CISupport/ews-build/events.py

```diff
@@ -56,7 +56,7 @@ class Events(service.BuildbotService):
         'run-layout-tests-without-change', 'layout-tests-repeat-failures-without-change',
         'run-layout-tests-in-stress-mode', 'run-layout-tests-in-guard-malloc-stress-mode',
         'run-api-tests', 'run-api-tests-without-change', 're-run-api-tests',
-        'scan-build', 'parse-static-analyzer-results', 'find-unexpected-results',
+        'find-unexpected-results', 'display-safer-cpp-results',
```

Comment:
What would happen in the case where scan-build failed and we didn’t get to those other productive steps?

Reply:
Oh, I see the status of the bubble is linked to these steps. I think it would be prudent to leave in `scan-build` then but `parse-static-analyzer-results` can be deleted.